scripts/base/frameworks/files/magic/general.sig

```diff
@@ -299,15 +299,27 @@ signature file-windows-minidump {
 
 # ISO 9660 disk image: First 16 sectors (2k) are arbitrary data.
 # The following sector is a volume descriptor with magic string "CD001"
-# at offset 1: 16 * 2048  + 1 = 32769
+# at offset 1: 16 * 2048 + 1 = 32769
 signature file-iso9660 {
         file-mime "application/x-iso9660-image", 99
-        file-magic /^.{32769}CD001/
+
+        # We know we expect the magic string at byte 34817, but cannot use a
+        # quantifier here since it would trigger state explosion in the regex
+        # engine, see #3622.
+        #
+        # TODO(bbannier): match for `/^.{34817}CD001/` once the regex engine is fixed.
```

Comment:
I'd be fine with commenting out the whole signature as the normal case won't work by default anyhow due to the limited bof buffer, leaving a comment that it can be done using separate scripts. Doesn't sound there is a good fix and the `.*` does result in the full bof being scanned for every file I believe.

Reply:
Benjamin, want me to pick this up, or you good with commenting them out completely, too?

I'm also thinking, we could place the signature and a `default_file_bof_buffer_size` redef into `policy/frameworks/signatures` to provide an easy (and functional) way for users to load it if they really want without having it in `base/`. It's not optimal, but seems better and more explicit than what we had in the past.

Reply:
Feel free to take this over. From an outside perspective I would have expected signatures to be independent from whatever `default_file_bof_buffer_size` was set to, and for signature matching to possibly provide gradually improve as more data becomes available.